src/other/Table.js

```diff
@@ -30,7 +30,7 @@
     Table.prototype.publish("fixedHeader", false, "boolean", "Enable or disable fixed table header",null,{tags:["Private"]});
     Table.prototype.publish("fixedColumn", false, "boolean", "Enable or disable fixed first column",null,{tags:["Private"]});
     
-    Table.prototype.publish("fixedSize", false, "boolean", "Fix Size to Min Width/Height");
+    Table.prototype.publish("fixedSize", true, "boolean", "Fix Size to Min Width/Height");
```

Comment:
This is what @jbrundage had it set to when he introduced it, I had changed it when I originally tried to fix this, but it broke it the table/legend in the Megachart.  I'm not sure what the default should be, or if it really makes a difference. Do you want me to change it? Any thoughts Jay?